coroutine/amd64/Context.S

```diff
@@ -57,4 +73,20 @@ PREFIXED_SYMBOL(SYMBOL_PREFIX,coroutine_transfer):
 
 #if (defined(__linux__) || defined(__FreeBSD__)) && defined(__ELF__)
 .section .note.GNU-stack,"",%progbits
+.section .note.gnu.property,"a"
```

Comment:
Is this still needed, given that the `cet.h` is included since d9398ac430062891185afe34b37dd293615af941?

Reply:
Yes, I had to remove that because it was incorrectly marking the file as supporting shadow-stacks when it in fact didn't. I think it's best to produce our own `.note.gnu.property` section here.

Reply:
https://github.com/ruby/ruby/blob/b811a9a097f2ce9ed9b200be84523698bc693e54/coroutine/amd64/Context.S#L8-L10